MCFM-JHUGen/src/Inc/spinzerohiggs_anomcoupl.f

```diff
@@ -111,6 +111,9 @@
       double complex dV_A,dP_A,dM_A,dFour_A
       double complex dV_Z,dP_Z,dM_Z,dFour_Z
       double complex dZZWpWm,dZAWpWm,dAAWpWm
+      double precision reZ,leZ,lnZ,rnZ
+      double precision clanou,cranou,clanod,cranod
+      integer AllowAnomalousZffCouplings
```

Comment:
Order of integer -> double precision -> double complex should be preserved.

Reply:
I also do not understand why you only deal with these couplings in this way. Implementation should be consistent with Z' couplings in JHUGen.